app/controllers/api/groups_controller.rb

```diff
@@ -56,4 +56,8 @@ def accessible_records
     Queries::ExploreGroups.new
   end
 
+  def resource_options(scope:, serializer:, root:)
```

Comment:
We don't currently serialize out any events for groups (it'd be the return values from methods in GroupService; so like `discussion_service.rb#create` returns an event because it calls `Events::NewDiscussion.publish!` as a return value, but `group_service.rb` doesn't ever do that.)

Doesn't mean we won't down the road though, which would be another gotcha.